src/MockClassifier.UnitTests/TokenServiceTest.cs

```diff
@@ -0,0 +1,46 @@
+using Xunit;
+using MockClassifier.Api.Services;
+using MockClassifier.Api.Models;
+using System;
+using System.Linq;
+using System.Collections.Generic;
+
+namespace MockClassifier.UnitTests
+{
+    public class TokenServiceTest
+    {
+        private readonly TokenService ministryClassifierService;
```

Comment:
nit: This is a super minor point, but I normally name the system under test as `sut` so it is clear throughout the tests which system is the one being tested vs mocks and other components. Feel free to ignore, it really does not matter. Discussion here FYI: https://stackoverflow.com/questions/7321407/what-is-sut-and-where-did-it-come-from#:~:text=System%20under%20test%20%28SUT%29%20refers%20to%20a%20system,%20tested%2C%20is%20called%20an%20application%20under%20test.